### DISCUSSION TITLE
Spatie Activity Logs on multiple database

### DISCUSSION THREAD

**Discussion by jvbalcita** [spatie/laravel-activitylog#874] [Q&A]
Just wanted to ask if there is a way that I can log a model depending on which database the model is assigned to.

For example I have DB1 for School1 Database and DB2 for School2 Database and I wanted to have a separate activity logs for each of the database using spatie package.

Not sure if that is possible.

Appreciate the response.

**Comment by Gummibeer** [comment]
It is - you would have to define the DB connection in a Middleware depending on which school this request is about.
Multi DB multi tenancy has several drawbacks and you should use a package for this and follow it's instructions.
